main/dhcp/src/EBox/DHCP.pm

```diff
@@ -738,6 +738,30 @@ sub fixedAddresses
     return $model->addresses($iface, $readOnly);
 }
 
+# Method: dhcpEnabled
+#
+#   Return dhcp interface state
+#
+# Parameters:
+#
+#   iface - String interface name
+#
+# Returns:
+#
+#   Boolean - true if dhcp is enabled for iface
+#
+# Exceptions:
+#
+
+sub dhcpEnabled
```

Comment:
About the method name, in order to follow the project's code style, it may be named "ifaceStatus".
And please, try to not to use ternary operators.